docs/AzureDevOps/GettingStarted.md

```diff
@@ -79,17 +79,19 @@ An example of a pipeline that builds and tests your PR can look like this:
 ```csharp
 ...
 
+private static readonly Variable s_DotnetVersionVariable = new Variable("DotnetVersion", string.Empty);
```

Comment:
I like this example but maybe we could just call it simply `DotnetVersion` to keep it nicer?

Reply:
done